src/lib.rs

```diff
@@ -1,58 +1,139 @@
+//! DLPack integration for Rust.
+//!
+//! This crate provides wrappers around the DLPack C API, allowing to exchange
+//! tensors with other libraries that support DLPack, such as PyTorch,
+//! TensorFlow, MXNet, CuPy, JAX, etc.
+//!
+//! The C API is available in the `sys` module, and the following higher-level
+//! wrappers are provided:
+//!
+//! - [`DLPackTensor`]: an owned DLPack tensor
+//! - [`DLPackTensorRef`]: a borrowed DLPack tensor (immutable)
+//! - [`DLPackTensorRefMut`]: a borrowed DLPack tensor (mutable)
+//!
+//! We also provide tools to convert from and to other rust types, using the
+//! `TryFrom` trait.
+//!
+//! ## Features
+//!
+//! - [`ndarray`]: enable conversion from and to `ndarray::Array`
+//! - [`pyo3`]: enable passing data from and to python, following the dlpack
+//!   protocol (the data is passed through a `PyCapsule` object).
```

Comment:
```suggestion
//! - [`pyo3`]: enable passing data from and to python, following the dlpack
//!             protocol (the data is passed through a `PyCapsule` object).
```

Reply:
I get this warning with this change: https://rust-lang.github.io/rust-clippy/master/index.html#doc_overindented_list_items